services/user/Branding/ui/src/App.tsx

```diff
@@ -0,0 +1,214 @@
+                <div
+                    className={
+                        uploadStatus === "Successful"
+                            ? "text-green-500"
+                            : "text-red-500"
+                    }
+                >
+                    {uploadStatus}
+                </div>
+                <div className="col-span-6 mt-6 grid grid-cols-6">
+                    <div className="col-span-2">
+                        <Label htmlFor="logoPath" className="block h-24 w-24">
+                            Logo
+                        </Label>
+                        <div className="text-extralight text-sm">
+                            Size: 96x96 px
+                        </div>
+                    </div>
+                    <div className="col-span-2">
+                        <img
+                            src={siblingUrl(
+                                null,
+                                "branding",
+                                "/network_logo.svg",
+                            )}
+                            height="96"
+                            width="96"
+                            className="b-1 col-span-3 h-24 w-24 object-cover"
+                            onClick={openFileClick}
+                        />
+                        <div className="text-extralight text-sm">
+                            Click logo to replace.
+                        </div>
+                    </div>
+                    <div
+                        className={
+                            "col-span-2" +
+                            (previewImgUrl.length ? "" : " invisible")
+                        }
+                    >
+                        <img
+                            src={previewImgUrl}
+                            height="96"
+                            width="96"
+                            className="block h-24 w-24 object-cover"
+                        />
+                        <div className="text-extralight align-bottom text-sm">
+                            Preview
+                        </div>
+                    </div>
+
+                    <Input
+                        id="logo-file"
+                        type="file"
+                        accept=".svg"
+                        className="invisible absolute"
+                        onChange={handleFileChange}
+                        ref={fileInput}
+                    />
+                </div>
+                <div className="col-span-6 mt-6 grid grid-cols-6">
+                    <Label htmlFor="networkName" className="col-span-2">
+                        Network name
+                    </Label>
+                    <Input
+                        id="networkName"
+                        className="col-span-4"
+                        onChange={(e) => {
+                            setNetworkName(e.target.value);
+                            setChangesMade(true);
```

Comment:
Once `changesMade` is set, it is never reset. Submitting successfully does not reset it. Changing the network name back to its original value doesn't reset it either.